index.html

```diff
@@ -94,17 +143,121 @@ <h3>Professional Art Printing Data</h3>
         </p>
         <ul class="languages">
           <li>css</li>
+          
           <li>html</li>
           <li>bootstrap</li>
           <li>ruby</li>
         </ul>
 
-        <button type="submit">See Project</button>
+        <button class="without-button" type="submit">See Project</button>
+      </div>
+    </div>
+
+    <div class="works-group-outside">
+      <div class="bg-page-one">
+        <div class="hidden-text">
+        <h3>Profesional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard.
+        </p>
+      
+        <ul class="languages">
+         
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+      </div>
+
+        <button  type="submit">See Project</button>
+      </div>
+    
+
+      <div class="bg-page-two">
```

Comment:
- [ ] Please when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image width